modules/300-prometheus/templates/prometheus/prometheus.yaml

```diff
@@ -134,7 +137,11 @@ spec:
   scrapeInterval: {{ .Values.prometheus.scrapeInterval | default "30s"}}
   evaluationInterval: {{ .Values.prometheus.scrapeInterval | default "30s" }}
   externalLabels:
+{{- if .Values.prometheus.externalLabels }}
+    {{- .Values.prometheus.externalLabels | toYaml | nindent 4 }}
+{{- else }}
     prometheus: deckhouse
```

Comment:
This label should always be present. Let's move it out of the condition. The only question is whether users want to override the value or not. 
🤔

Reply:
btw, I found that we use `prometheus=deckhouse` ubiquitously for alerts grouping [like](https://github.com/deckhouse/deckhouse/blob/d171e9edcc6a44dc6bf2254e9df2fae104da6543/modules/300-prometheus/monitoring/prometheus-rules/grafana.yaml#L18). Maybe it's better to leave it as it is without a way to change it value.

Reply:
Oh, good catch. Yes, you are right.